react/admin/CustomField.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Input } from 'vtex.styleguide'
+
+import type { CustomField } from './CustomFields'
+
+interface CustomFieldProps {
+  index: number
+  handleUpdate: (index: number, customField: CustomField) => void
+  fieldLabel: string
+  fieldValue: string
+  fieldType: 'text'
+  isDefaultCustomField?: boolean
```

Comment:
✅ if located inside Custom Fields tab, where we setup default fields for all organizations, update name of the custom field. In case located inside Organizations details, update value of the field

Reply:
Ah, I understand it now.